source/includes/_embed2.md.erb

```diff
@@ -375,6 +375,11 @@ When called, the `zdChatterAuthCallback` can be used to request a JWT token from
 ## Actions
 Actions can be called from the global adaEmbed object. A list of the available actions is available below.
 
+### closeCampaign
+`closeCampaign(): Promise<void>;`
```

Comment:
We need the disclaimer that this is only for engage clients.

Reply:
@GuelorEmanuel ☝️ 
i think Andrei's talking about the disclaimer from line 530

Reply:
I see, thanks for pointing that out.